Provider/src/EntityFramework.Firebird/SqlGen/SqlGenerator.cs

```diff
@@ -2650,19 +2650,60 @@ private static ISqlFragment HandleCanonicalFunctionDateTimeDiff(SqlGenerator sql
 		private static ISqlFragment HandleCanonicalFunctionCreateDateTime(SqlGenerator sqlgen, DbFunctionExpression e)
 		{
 			var result = new SqlBuilder();
+
 			result.Append("CAST('");
-			result.Append(e.Arguments[0].Accept(sqlgen));
+
+			if (e.Arguments[0].ExpressionKind==DbExpressionKind.Constant) // year
```

Comment:
Spaces around `==` (and other places).

Reply:
Please update/complete your `.editorconfig` file. Export your current VS formatting settings (it's a no-brainer done in a minute) or have a look at https://github.com/cincuranet/FirebirdSql.Data.FirebirdClient/pull/94. So VS will take care of it.